examples/community/README.md

```diff
@@ -3741,3 +3741,236 @@ onestep_image = pipe(prompt, num_inference_steps=1).images[0]
+| Using IP-Adapter with negative noise                                                                                                  | Using negative noise with IP-adapter to better control the generation (see the [original post](https://github.com/huggingface/diffusers/discussions/7167) on the forum for more details)                                                                                                                                                                                                                                                    | [IP-Adapter Negative Noise](#ip-adapter-negative-noise)                                   | | [Álvaro Somoza](https://github.com/asomoza)|
+| asymmetric tiling                                                                                                  |configure seamless image tiling independently for the X and Y axes                                                                                                                                                                                                      | [Asymmetric Tiling](#asymmetric-tiling )                                   | | [alexisrolland](https://github.com/alexisrolland)|
+
+
+## Example usages
+
+### IP Adapter Negative Noise
+
+Diffusers pipelines are fully integrated with IP-Adapter, which allows you to prompt the diffusion model with an image. However, it does not support negative image prompts the same way it supports negative text prompts: it does not accept a `negative_ip_adapter_adapter` argument; when you pass an `ip_adapter_image,` it will create a zero-filled tensor as a negative image. This script shows you how to create a negative noise from `ip_adapter_image` and use it to significantly improve the generation quality while preserving the composition of images.
+
+[cubiq](https://github.com/cubiq) initially developed this feature in his [repository](https://github.com/cubiq/ComfyUI_IPAdapter_plus). The community script was contributed by [asomoza](https://github.com/Somoza). You can find more details about this experimentation [this discussion](https://github.com/huggingface/diffusers/discussions/7167)
```

Comment:
How about providing a teaser along with results. Let's maybe not include code because the scripts themselves will provide them. 

Also, WDYT about having a separate `README_community_scripts.md` to keep the current README a little less cluttered and easy to follow?

Reply:
➕ 1 for having a separate `README.md` for the community scripts!

Reply:
@sayakpaul 
addressed the separate README part!

But I left the codes there for now because I did not make a separate `.py` file for each - I feel like these are just tips and tricks that would be used with some other things so it might be easier in `md` format so user can just copy-paste it

it's not ideal because they are a bit long so feel free to help improve it with a PR if you have good ideas!